addons/advanced_throwing/functions/fnc_throw.sqf

```diff
@@ -49,9 +49,15 @@ if (!(_unit getVariable [QGVAR(primed), false])) then {
         _newVelocity = _newVelocity vectorAdd (velocity (vehicle _unit));
     };
 
+    private _config = configFile >> "CfgAmmo" >> typeOf _activeThrowable;
+    private _torqueDir = getArray (_config >> QGVAR(torqueDirection));
+    private _torqueMag = getNumber (_config >> QGVAR(torqueMagnitude));
+    private _torque = vectorNormalized (_torqueDir vectorMultiply _torqueMag);
```

Comment:
Why is torqueMagnitude a complex random number if we normalize it?
`(50 + random 100)`  - 50 or 150 would produce the same end value

Reply:
@commy2 ?

Reply:
Because this change is wrong and doesn't do what the previous code did.
Should be:
```sqf
    private _torqueDir = vectorNormalized getArray (_config >> QGVAR(torqueDirection));
    private _torqueMag = getNumber (_config >> QGVAR(torqueMagnitude));
    private _torque = _torqueDir vectorMultiply _torqueMag;
```

Reply:
Oh, my bad! 😆